source/views/main.js

```diff
@@ -162,7 +162,7 @@ create: function(){
 	this.haveCompass = false;
 	this.compassOpacity = 0;
 	/* Adaptation to Firefox/FirefoxOS */
-	this.transform = (enyo.platform.firefoxOS || enyo.platform.platformName === "firefox") ? "" : "-webkit-";
+	this.transform = ""; //(enyo.platform.firefoxOS || enyo.platform.platformName === "firefox") ? "" : "-webkit-";
```

Comment:
Current WebKit/Blink allows for "transform" use without the "-webkit-" prefix.  But this does still work without this, so that's fine; this can be undone.